### ISSUE TITLE
Texte dans la boite de dialogue

### ISSUE THREAD

**Issue by raphael-g** [loult-elte-fwere/loult-ng#81]
Lorsque le texte est suffisamment long pour dépasser la largeur de la boite de dialogue, celui-ci passe en hors champ en  au lieu de rester à la vue de l'utilisateur (en principe c'est la partie gauche du texte qui passe en hors champ à gauche sur une boite qui fonctionne correctement).

**Comment by hadware** [comment]
yes j'ai remarqué ça, je me demandais si c'était que chez moi. Visiblement c'est un truc qui a changé suite à une MAJ sur FireCroûte non? T'as une idée de comment fixer ça?
